cms/models.py

```diff
@@ -90,4 +90,25 @@ class CoursePage(ProductPage):
         on_delete=models.SET_NULL,
         help_text="The course for this page",
     )
-    content_panels = [FieldPanel("course")] + ProductPage.content_panels
+    video_url = models.URLField(
+        null=True,
+        blank=True,
+        help_text="URL to the video to be displayed for this course"
+    )
+    video_title = RichTextField(
+        blank=True, help_text="The title to be displayed for the course video"
+    )
+    background_image = models.ForeignKey(
+        Image,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
```

Comment:
any particular reason to define `related_name` as `+`?

Reply:
We don't need a reverse for this. Avoiding name conflicts further down the road.